libuavcan/include/uavcan/bus/can.hpp

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (C) 2014 Pavel Kirienko <pavel.kirienko@gmail.com>
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ */
+/** @file */
+
+#ifndef UAVCAN_BUS_CAN_HPP_INCLUDED
+#define UAVCAN_BUS_CAN_HPP_INCLUDED
+
+#include <array>
+
+#include "uavcan/uavcan.hpp"
+
+namespace uavcan
+{
+/**
+ * @namespace bus
+ * Contains bus-specific namespaces.
+ */
+namespace bus
+{
+/**
+ * @namespace CAN
+ * Types for working with UAVCAN on a Controller Area Network.
+ */
+namespace CAN
+{
+/**
+ * @namespace TypeFd
+ * Properties of an ISO compliant CAN-FD bus.
+ */
+namespace TypeFd
+{
+/**
+ * The maximum size of a data frame for this bus.
+ */
+constexpr static size_t max_frame_size_bytes = 64;
```

Comment:
Bad naming, should be PascalCase.

Should those not be contained inside a struct for easier grouping when used in templates?